CSE437s/FL24-project-group-9
# Implement BFF API and Set up Poetry

# Description

This PR implement basic API for the FE and set up Poetry to manage package in BE.
Refer [here](https://dbdiagram.io/d/WashU-Course-Scheduler-66f5e7083430cb846cc45fbe) for the current schema.

## Related PRs

n/a

## Type of change

### Please delete options that are not relevant


## Screenshots/Recordings

<img width="1512" alt="Screenshot 2024-09-27 at 11 53 18" src="https://github.com/user-attachments/assets/049835af-0b78-4aa4-bb09-0e230e4be5a8">

## Testing instructions

Instructions on how to test the changes made in the PR, helping reviewers validate the code.

1. Run `git checkout branch_name`.
2. Run `poetry install` and `poetry run python3 manage.py runserver` to run the server.
3. Navigate to '/api/' to see all available endpoints.

## Notes

I will update `README.md` in the next PR.
